src/flags.jl

```diff
@@ -0,0 +1,226 @@
+# Flags
+# =========
+#
+
+"""
+    flag(record::XAMRecord})::UInt16
+
+Get the bitwise flags of `record`.
+The returned value is a `UInt16` of each flag being OR'd together.
+The possible flags are:
+
+    0x0001 template having multiple segments in sequencing
+    0x0002 each segment properly aligned according to the aligner
+    0x0004 segment unmapped
+    0x0008 next segment in the template unmapped
+    0x0010 SEQ being reverse complemented
+    0x0020 SEQ of the next segment in the template being reverse complemented
+    0x0040 the first segment in the template
+    0x0080 the last segment in the template
+    0x0100 secondary alignment
+    0x0200 not passing filters, such as platform/vendor quality controls
+    0x0400 PCR or optical duplicate
+    0x0800 supplementary alignment
+"""
+
+function flag end
```

Comment:
I would call these `flags`, in plural, since it gives them all, but no big deal.

Reply:
That might help. Would you also change the `record.flag` field name to `record.flags`?

Reply:
Yeah

Reply:
Resolved by e8cfcf9.